build-plugin/action.yml

```diff
@@ -88,6 +92,12 @@ runs:
       run: if [ "v${{ steps.package-plugin.outputs.plugin-version }}" != "${{ steps.metadata.outputs.github-tag }}" ]; then printf "\033[0;31mPlugin version doesn't match tag name. The tag should be v${{ steps.package-plugin.outputs.plugin-version }} \033[0m\n"; exit 1; fi
       shell: bash
 
+    - name: Generate artifact attestation
+      if: ${{ inputs.attestation == 'true' }}
+      uses: actions/attest-build-provenance@v1
```

Comment:
Is there a specific reason why we are using `v1` instead of `v2`? https://github.com/actions/attest-build-provenance/releases

Reply:
not really. just didn't check there was  v2.